spec/requests/delayed_job_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "Delayed Job web interface", type: :request do
+  let(:user)          { create(:user) }
+  let(:super_admin)   { create(:user, :super_admin) }
+  let(:article)       { create(:article, user_id: user.id) }
+
+  describe "GET /delayed_job" do
+    context "when not logged in" do
+      it "redirects to /enter" do
+        expect do
+          get "/delayed_job"
+        end.to raise_error(ActionController::RoutingError)
+      end
+    end
+
+    context "when logged in" do
+      it "redirects to /ente" do
```

Comment:
just a small typo here

Reply:
Oops, somehow commited the unfinished version. Thanks!